portal-kernel/src/com/liferay/users/admin/kernel/util/UsersAdminUtil.java

```diff
@@ -120,221 +120,227 @@ public static void addPortletBreadcrumbEntries(
 		throws PortalException {
 
-		return getUsersAdmin().getOrganizations(hits);
+		return _usersAdmin.getOrganizations(hits);
 	}
 
 	public static List<OrgLabor> getOrgLabors(ActionRequest actionRequest) {
-		return getUsersAdmin().getOrgLabors(actionRequest);
+		return _usersAdmin.getOrgLabors(actionRequest);
 	}
 
 	public static List<Phone> getPhones(ActionRequest actionRequest) {
-		return getUsersAdmin().getPhones(actionRequest);
+		return _usersAdmin.getPhones(actionRequest);
 	}
 
 	public static List<Phone> getPhones(
 		ActionRequest actionRequest, List<Phone> defaultPhones) {
 
-		return getUsersAdmin().getPhones(actionRequest, defaultPhones);
+		return _usersAdmin.getPhones(actionRequest, defaultPhones);
 	}
 
 	public static long[] getRoleIds(PortletRequest portletRequest)
 		throws PortalException {
 
-		return getUsersAdmin().getRoleIds(portletRequest);
+		return _usersAdmin.getRoleIds(portletRequest);
 	}
 
 	public static OrderByComparator<Role> getRoleOrderByComparator(
 		String orderByCol, String orderByType) {
 
-		return getUsersAdmin().getRoleOrderByComparator(
-			orderByCol, orderByType);
+		return _usersAdmin.getRoleOrderByComparator(orderByCol, orderByType);
 	}
 
 	public static <T> String getUserColumnText(
 		Locale locale, List<? extends T> list, Accessor<T, String> accessor,
 		int count) {
 
-		return getUsersAdmin().getUserColumnText(locale, list, accessor, count);
+		return _usersAdmin.getUserColumnText(locale, list, accessor, count);
 	}
 
 	public static long[] getUserGroupIds(PortletRequest portletRequest)
 		throws PortalException {
 
-		return getUsersAdmin().getUserGroupIds(portletRequest);
+		return _usersAdmin.getUserGroupIds(portletRequest);
 	}
 
 	public static OrderByComparator<UserGroup> getUserGroupOrderByComparator(
 		String orderByCol, String orderByType) {
 
-		return getUsersAdmin().getUserGroupOrderByComparator(
+		return _usersAdmin.getUserGroupOrderByComparator(
 			orderByCol, orderByType);
 	}
 
 	public static List<UserGroupRole> getUserGroupRoles(
 			PortletRequest portletRequest)
 		throws PortalException {
 
-		return getUsersAdmin().getUserGroupRoles(portletRequest);
+		return _usersAdmin.getUserGroupRoles(portletRequest);
 	}
 
 	public static List<UserGroup> getUserGroups(Hits hits)
 		throws PortalException {
 
-		return getUsersAdmin().getUserGroups(hits);
+		return _usersAdmin.getUserGroups(hits);
 	}
 
 	public static OrderByComparator<User> getUserOrderByComparator(
 		String orderByCol, String orderByType) {
 
-		return getUsersAdmin().getUserOrderByComparator(
-			orderByCol, orderByType);
+		return _usersAdmin.getUserOrderByComparator(orderByCol, orderByType);
 	}
 
 	public static List<User> getUsers(Hits hits) throws PortalException {
-		return getUsersAdmin().getUsers(hits);
+		return _usersAdmin.getUsers(hits);
 	}
 
+	/**
+	 * @deprecated As of Athanasius (7.3.x)
+	 */
+	@Deprecated
 	public static UsersAdmin getUsersAdmin() {
 		return _usersAdmin;
 	}
```

Comment:
I think since the only real change here is _how_ the dependency gets injected, we don't need to deprecate this method yet (unless there's another reason that I'm missing).